cfpb/consumerfinance.gov
# P4C: Circle colors should match gas gauges

The repayment and graduation rate circle colors should match the gas gauge color.


## Additions

- Set class for repayment and graduation rate circles when the gas gauge chart is set up
- Make default rate circle gray since ranking data is not available


## How to test this PR

1. Check out branch
2. Navigate to [financial path tool](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/?iped=174428&houp=offCampus&typp=associates&lenp=2&ratp=inState). On Compare to other schools tab, note that the percentage circle colors match the gas gauge colors.
3. Navigate to [same scenario](https://www.consumerfinance.gov/paying-for-college/your-financial-path-to-graduation/?iped=174428&houp=offCampus&typp=associates&lenp=2&ratp=inState&depp=independent&cobs=n&tuit=5791&hous=8260&book=1000&pers=15051&houx=1013&fdx=417&clhx=86&trnx=569&hltx=299&entx=151&retx=196&taxx=77&othx=258) in production and note that the colors do not always match

4. Verify that default rate circle is gray


## Screenshots

### Branch
<img width="665" alt="Screen Shot 2021-02-01 at 8 43 21 AM" src="https://user-images.githubusercontent.com/778171/106489448-a48d4f00-6469-11eb-8731-00349ba12cd1.png">


### Production
<img width="653" alt="Screen Shot 2021-02-01 at 8 43 32 AM" src="https://user-images.githubusercontent.com/778171/106489472-a9ea9980-6469-11eb-8da8-8595ab62c7f8.png">

### Default rate
<img width="661" alt="gray-circle" src="https://user-images.githubusercontent.com/778171/109039559-c604e380-7681-11eb-9175-a95abf566e47.png">




## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
